### PR TITLE
Debian package improvements

### PR DESCRIPTION
**Pull request by philippmeisberger**

### CODE REVIEW

File: src/debian/python-fingerprint.install [hunk 1]
```diff
@@ -1 +0,0 @@
-usr/lib/python2*/*-packages/*
```
Comment (bastianraschke) [inline src/debian/python-fingerprint.install:1]: Why not needed anymore?

File: src/debian/rules [hunk 2]
```diff
@@ -1,20 +1,10 @@
...
+
 %:
-	dh $@ --with python2,python3
+	dh $@ --with python2,python3 --buildsystem=pybuild
 
 override_dh_auto_clean:
 	dh_auto_clean
 	rm -rf ./build/ ./files/*.egg-info/
-
-override_dh_auto_build:
```
Comment (bastianraschke) [inline src/debian/rules]: This is done via `pybuild` buildsystem?

File: src/setup.py [hunk 3]
```diff
@@ -3,10 +3,15 @@
...
+import sys
+sys.path.insert(0, './files/')
+
+import pyfingerprint
+
 setup(
     name            = 'pyfingerprint',
-    version         = '1.5', ## Never forget to change module version as well!
-    description     = 'Python written library for using the ZFM-20 fingerprint sensor.',
+    version         = pyfingerprint.__version__,
```
Comment (bastianraschke) [inline src/setup.py:13]: :+1:

**Comment by bastianraschke** [comment]
Please fix conflicts

File: src/debian/python-fingerprint.install (hunk 1 above)
Comment (philippmeisberger) [reply src/debian/python-fingerprint.install:1]: This is done by pybuild

File: src/debian/rules (hunk 2 above)
Comment (philippmeisberger) [reply src/debian/rules]: Yes

**Review by bastianraschke — APPROVED** [review]

**Pull request merged**